packages/core/src/models/Manifest.ts

```diff
@@ -9,8 +9,8 @@ class ApiEndPointType extends t.Type<string> {
       (input, context) => {
         if (typeof input !== "string") return t.failure(input, context);
         const regx = new RegExp(
-          "^https?://[a-z0-9][a-z0-9-]{2,}[a-z0-9].(prismic.io|wroom.io|wroom.test)/api/v2/?$",
-          "i"
+          "^https?://[a-z0-9][a-z0-9-]{2,}[a-z0-9](.cdn)?.(prismic.io|wroom.io|wroom.test)/api/v2/?$",
```

Comment:
I've tested the Regexp online and it seems that the regexp now returns 2 groups: `cdn` and `prismic.io`.
How do you know that your function will return the second parameter now ?

Reply:
it users the the `test` method, it only returns true or false